rule-example.yaml

```diff
@@ -241,4 +241,28 @@
       - beans.xml
       namespaces:
         b: http://xmlns.jcp.org/xml/ns/javaee
-      xpath: /b:beans
\ No newline at end of file
+      xpath: /b:beans
+- category: potential
+  customVariables: []
```

Comment:
nit: can we shrink this rule, just keep the required fields for brevity i.e. remove labels, links, customVariables?